_data/compatibility/blocksettle.yaml

```diff
@@ -0,0 +1,96 @@
+---
+name: BlockSettle
+internal_url: /en/compatibility/blocksettle
+logo: /img/compatibility/blocksettle/blocksettle.png
+rbf:
+  tested:
+    date: "2020-10-29"
+    platforms:
+      - Windows
+      - macOS
+      - Linux
+    version: "0.91.1"
+  features:
+    receive:
+      notification: "true"
+      list: "true"
+      details: "true"
+      shows_replaced_version: "true"
+      shows_original_version: "false"
+    send:
+      signals_bip125: "true"
+      list: "true"
+      details: "false"
+      shows_replaced_version: "true"
```

Comment:
Is there a screenshot that shows this distinction between shows_replaced_version vs shows_original_version ?

Reply:
We have only a replaced TX shown, "shows_original_version" is set to "false" in both cases.